### PR TITLE
docs: add porch ui installation guide

### PR DESCRIPTION
**Pull request by ChristopherFry**
Adds the Configuration as Data UI (also known as Porch UI) installation guide.

### CODE REVIEW

File: site/guides/porch-ui-installation.md
Comment (bgrant0607) [inline site/guides/porch-ui-installation.md:12]: Link directly to the README?

File: site/guides/porch-ui-installation.md
Comment (bgrant0607) [inline site/guides/porch-ui-installation.md:24]: To access the UI or run the UI?

File: site/guides/porch-ui-installation.md
Comment (bgrant0607) [inline site/guides/porch-ui-installation.md:9]: Added some subsections would be useful.

## Installing in Kubernetes
...

## Running locally in a container
...

File: site/guides/porch-ui-installation.md
Comment (ChristopherFry) [reply site/guides/porch-ui-installation.md]: Added.

File: site/guides/porch-ui-installation.md
Comment (ChristopherFry) [reply site/guides/porch-ui-installation.md]: Updated header.

File: site/guides/porch-ui-installation.md
Comment (ChristopherFry) [reply site/guides/porch-ui-installation.md]: Added `Running in Backstage` subsection since at this point, the plugin can be configured to be ran locally or within Kubernetes. I added this section after 'Running locally in a container' since adding the plugin to Backstage is more time consuming process.

File: site/guides/porch-ui-installation.md
Comment (droot) [inline site/guides/porch-ui-installation.md:1]: I am wondering if we should refer the UI as `Package Orchestrator UI` ?

File: site/guides/porch-ui-installation.md
Comment (droot) [inline site/guides/porch-ui-installation.md:26]: nit: may be remove kpt-backstage-plugins dir in the end ?

File: site/guides/porch-ui-installation.md
Comment (droot) [inline site/guides/porch-ui-installation.md:52]: Can we add link to this guide in `installation/README.md` so that all components are covered.

**Review by droot — APPROVED** [review]
Some minor comments.

File: site/guides/porch-ui-installation.md
Comment (ChristopherFry) [reply site/guides/porch-ui-installation.md:1]: @bgrant0607 Thoughts?

File: site/guides/porch-ui-installation.md
Comment (ChristopherFry) [reply site/guides/porch-ui-installation.md]: Removed.

File: site/guides/porch-ui-installation.md
Comment (ChristopherFry) [reply site/guides/porch-ui-installation.md:1]: The `Configuration as Data` name I have already started publishing with, so unless we will be only referring to the UI as the `Package Orchestrator` within kpt.dev, renaming is starting to get to the point where it becomes difficult (although not impossible).

Comment (bgrant0607) [reply site/guides/porch-ui-installation.md:1]: "Package Orchestrator" is not going to be a thing we advertise to users. It's a component. It's relevant to contributors. Let's stay with "Configuration as Data" for now. That's the overall approach, vs IaC. And "kpt" is the overall project name, so that would be the other option.

Comment (bgrant0607) [reply site/guides/porch-ui-installation.md:1]: I know we have the user guide discussing porch for now, that's fine.

File: site/guides/porch-ui-installation.md
Comment (ChristopherFry) [reply site/guides/porch-ui-installation.md:52]: Added.

**Review by bgrant0607 — APPROVED** [review]
LGTM

**Comment by ChristopherFry** [comment]
@bgrant0607 @droot Can one of you merge this pull request once CI completes? I cannot merge this myself since i don't have write access to the repository. Thanks!

File: site/guides/porch-ui-installation.md
Comment (droot) [reply site/guides/porch-ui-installation.md:1]: Ack.

File: site/guides/porch-ui-installation.md
Comment (droot) [reply site/guides/porch-ui-installation.md:52]: Thanks

**Pull request merged**
